app/controllers/giveaways_controller.rb

```diff
@@ -55,10 +55,6 @@ def edit
     def show
         @giveaway = Giveaway.find(params[:id])
         if !@giveaway.isActive
```

Comment:
Ah sorry, should have noticed this is an attribute. In db, I would name it just `active`. The `?` is for boolean methods, but don't put them in the db :D my bad